paystack-domain/src/main/java/com/chrisworks/paystackclient/domain/product/UpdateProductRequest.java

```diff
@@ -0,0 +1,11 @@
+package com.chrisworks.paystackclient.domain.product;
+
+import com.chrisworks.paystackclient.domain.Amount;
+
+public final class UpdateProductRequest extends CreateProductRequest {
+
```

Comment:
Re-use the `CreateProductRequest,` maybe rename it to `CreateOrUpdateProductRequest`

Reply:
Done